packages/std/src/traits.rs

```diff
@@ -246,3 +250,29 @@ impl<'a> QuerierWrapper<'a> {
         Ok(res.delegation)
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use crate::mock::MockQuerier;
+
+    // this is a simple demo helper to prove we can use it
+    fn demo_helper(_querier: &dyn Querier) -> u64 {
+        2
+    }
+
+    // this just needs to compile to prove we can use it
+    #[test]
+    fn use_querier_wrapper_as_querier() {
+        let querier: MockQuerier<Empty> = MockQuerier::new(&[]);
+        let wrapper = QuerierWrapper::new(&querier);
+
+        // call with deref shortcut
+        let res = demo_helper(&*wrapper);
+        assert_eq!(2, res);
+
+        // call with explicit deref
+        let res = demo_helper(wrapper.deref());
+        assert_eq!(2, res);
```

Comment:
Could you add a test calling `wrapper.query_raw` to see if we can access it without exmplicit deref?